### PR TITLE
Darinkrauss/platform endpoints

### PR DESCRIPTION
**Pull request by jh-bate**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -351,6 +353,104 @@ module.exports = function (config, deps) {
...
+        });
+    },
+    /**
+     * Finalize the given dataset (close and post-process)
+     *
+     * @param {String} datasetId of the dataset to finalize
+     * @param cb
+     * @returns {cb}  cb(err, response)
+     */
+    finalizeDataset: function (datasetId, cb) {
```
Comment (jh-bate) [inline index.js]: `datasetId`  vs `dataSetId` below

**Comment by jh-bate** [comment]
@gniezen one tiny nitpick other than that looking good

File: index.js (hunk 1 above)
Comment (gniezen) [inline index.js]: Fixed.

**Comment by gniezen** [comment]
Merging.

**Pull request merged**